.cirrus.yml

```diff
@@ -2,15 +2,16 @@ task:
   name: "FreeBsd 12.0 amd64  [GOAL: install]  [no depends, only system libs]"
   freebsd_instance:
     image: freebsd-12-0-release-amd64
-  ccache_cache:
-    folder: "/tmp/ccache_dir"
   env:
     MAKEJOBS: "-j3"
     CONFIGURE_OPTS: "--disable-dependency-tracking"
     GOAL: "install"
     CCACHE_SIZE: "200M"
     CCACHE_COMPRESS: 1
     CCACHE_DIR: "/tmp/ccache_dir"
+  ccache_cache:
+    folder: "/tmp/ccache_dir"
+  timeout_in: 80m
```

Comment:
If you are hitting the time out limit but your build can benefit from more CPUs, you can try to increase VM resources to the maximum allowed:

```yaml
freebsd_instance:
    image: freebsd-12-0-release-amd64
    cpu: 8
    memory: 24G
```